src/indexers/vault-harvests-indexer.ts

```diff
@@ -7,9 +9,11 @@ export async function refreshVaultHarvests() {
   const chains = loadChains();
   await Promise.all(
     chains.map(async (chain) => {
+      const graph = new BadgerGraph({ network: chain.network });
```

Comment:
is there a reason to do one more import?
`chain.getSdk().graph` should be available